### PR TITLE
New package: Multitail

### PR DESCRIPTION
**Pull request by fronald**
MultiTail lets you view one or multiple files like the original tail program. The difference is that it creates multiple windows on your console (with ncurses).

More details: https://vanheusden.com/multitail/

image

### CODE REVIEW

File: components/sysutils/multitail/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/multitail
+file path=usr/etc/multitail.conf.new
+file path=usr/etc/multitail/conversion-scripts/colors-example.pl
```
Comment (AndWac) [inline components/sysutils/multitail/manifests/sample-manifest.p5m]: usr/etc is wrong.

Comment (fronald) [reply components/sysutils/multitail/manifests/sample-manifest.p5m]: But this is the sample (generated by gmake). The effective manifest is multitail.p5m.

Comment (AndWac) [reply components/sysutils/multitail/manifests/sample-manifest.p5m]: Ok, but if the manifest uses a wrong folder for the configuration it's quite likely that the real application is also looking at the wrong path for its configuration. You should use settings in the Makefile to fix this.

Comment (fronald) [reply components/sysutils/multitail/manifests/sample-manifest.p5m]: Nice. Done :-D

**Comment by fronald** [comment]
sample-manifest fixed.

**Comment by AndWac** [comment]
You shall not edit the sample-manifest. It's not possible to fix this issue with that. You'll need to change the Makefile (or the sources by patches) to make a gmake sample-manifest generate the expected results.

**Comment by fronald** [comment]
> You shall not edit the sample-manifest. It's not possible to fix this issue with that. You'll need to change the Makefile (or the sources by patches) to make a gmake sample-manifest generate the expected results.

Oh sorry. I was always thinking "what is this sample-manifest for?", now I understand the purpose... It is to validate the installation prefixes of the construction stage.

I'll fix it. Thanks for the answer.

**Comment by AndWac** [comment]
The sample-manifest is the basis to build the real manifest from. Sometimes you can use the sample-manifest by just copy it , sometimes it needs some specialities for IPS.
Furthermore, with the sample-manifest you can check for differences when you rebuild or update to a newer version. You can see the differences easier than comparing a new sample-manifest against the adopted manifest.

**Comment by fronald** [comment]
> The sample-manifest is the basis to build the real manifest from. Sometimes you can use the sample-manifest by just copy it , sometimes it needs some specialities for IPS. Furthermore, with the sample-manifest you can check for differences when you rebuild or update to a newer version. You can see the differences easier than comparing a new sample-manifest against the adopted manifest.

Thank you for the explanation.

I have already changed the Makefile to prepare the prototype directory with the correct Multitail installation artifacts.

I have already checked that multitail is seeing the configuration file correctly after it is installed (/etc/multitail.conf), as can be seen in CMakeLists.txt in the value of the "CONFIG_FILE" property.

Done \o/

**Comment by AndWac** [comment]
Please check the sample-manifest again. mine differs.

**Comment by fronald** [comment]
> Please check the sample-manifest again. mine differs.

Done.

**Pull request merged**